### PR TITLE
Added detailed guide about DipDup indexer

### PR DESCRIPTION
**Pull request by isfriz**

### CODE REVIEW

File: docs/developers/tooling/data-indexers/dipdup/overview.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: Overview
+sidebar_position: 1
+image: /img/socialCards/dipdup.jpg
+---
+
+[DipDup](https://dipdup.io/) is a Python framework for building smart contract indexers. It helps developers focus on business logic instead of writing a boilerplate to store and serve data. DipDup-based indexers are selective, which means only required data is requested. This approach allows to achieve faster indexing times and decreased load on underlying APIs.
+
+This page will guide you through the steps to get your first DipDup indexer up and running in a few minutes without getting too deep into the details.
```
Comment (jlwllmr) [inline docs/developers/tooling/data-indexers/dipdup/overview.md:9]: It feels like lines 9-11 belong more in the quickstart file as an intro than they do here?

**Comment by jlwllmr** [comment]
@isfriz nice work - just the one bit of feedback in a comment, but it's not essential. 

Before merging:
- A redirect from the old path `/data-indexers/dipdup` will be necessary. There isn't an index file in the new `/dipdup` directory you created, so any user visiting the old path will 404. I suggest you add a redirect from the old path to the overview page. Redirects are in `docusuarus.config.js`
- Please add `pipx` to the `project-words.txt` to pass the linter. 

Thanks!

**Comment by isfriz** [comment]
> @isfriz nice work - just the one bit of feedback in a comment, but it's not essential.
> 
> Before merging:
> 
> * A redirect from the old path `/data-indexers/dipdup` will be necessary. There isn't an index file in the new `/dipdup` directory you created, so any user visiting the old path will 404. I suggest you add a redirect from the old path to the overview page. Redirects are in `docusuarus.config.js`
> * Please add `pipx` to the `project-words.txt` to pass the linter.
> 
> Thanks!

Thx 4 review! 
Done!

**Review by jlwllmr — APPROVED** [review]
LGTM

**Pull request merged**